<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1050</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1050</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow { background-image: url(:/images/background/background_purple.png); }
QGroupBox {
border: 2px solid #aa55ff;
border-radius: 15px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelToDoList">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>170</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 10pt &quot;Merriweather&quot;;</string>
    </property>
    <property name="text">
     <string>To Do List</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>350</width>
      <height>210</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 15pt &quot;Merriweather&quot;;
</string>
    </property>
    <property name="title">
     <string>Login</string>
    </property>
    <widget class="QLabel" name="labelPassword">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>75</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Merriweather&quot;;</string>
     </property>
     <property name="text">
      <string>Password :</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelUsername">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>35</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Merriweather&quot;;</string>
     </property>
     <property name="text">
      <string>Username :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditUsername">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>38</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
font: 9pt&quot;Merriweather&quot;;
border-radius: 7px;
background-color: rgb(215, 200, 255);
border: 1px solid #000000;
padding: 5px;
}
QLineEdit:hover {
	background-color: rgb(206, 192, 247);
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPassword">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>78</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
font: 9pt&quot;Merriweather&quot;;
border-radius: 7px;
background-color: rgb(215, 200, 255);
border: 1px solid #000000;
padding: 5px;
}
QLineEdit:hover {
	background-color: rgb(206, 192, 247);
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelError">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Merriweather&quot;;
color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>error error error error error</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonShow">
     <property name="geometry">
      <rect>
       <x>266</x>
       <y>78</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/icon/eye_icon.png</normaloff>
       <normalon>:/images/icon/not_eye_icon.png</normalon>:/images/icon/eye_icon.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLogin">
     <property name="geometry">
      <rect>
       <x>131</x>
       <y>134</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover {
  background-color: rgb(165, 125, 235);
}
QPushButton {
	font: 10pt&quot;Merriweather&quot;;
	border-radius: 7px;
	background-color: rgb(180, 135, 255);
	border: 1px solid #000000;
	color: rgb(0, 0, 0);
}
QPushButton:pressed {
	background-color: rgb(135, 100, 195);
}</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <zorder>lineEditUsername</zorder>
    <zorder>labelPassword</zorder>
    <zorder>lineEditPassword</zorder>
    <zorder>labelError</zorder>
    <zorder>labelUsername</zorder>
    <zorder>pushButtonShow</zorder>
    <zorder>pushButtonLogin</zorder>
   </widget>
   <widget class="QLabel" name="labelWelcomeBack">
    <property name="geometry">
     <rect>
      <x>398</x>
      <y>120</y>
      <width>380</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 30pt &quot;Merriweather&quot;;</string>
    </property>
    <property name="text">
     <string>Welcome Back !</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSignup">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>450</y>
      <width>230</width>
      <height>29</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 10pt &quot;Merriweather&quot;;
	text-decoration: underline;
	color: rgb(0, 0, 0);
	background: transparent;
}
QPushButton:hover {
	color: rgb(68, 0, 102);
	font: 10.5pt &quot;Merriweather&quot;;
}
QPushButton:pressed {
	color: rgb(23, 0, 36);
}</string>
    </property>
    <property name="text">
     <string>Don't Have Any Account?</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
